llm/mixtral/serve.yaml

```diff
@@ -14,7 +14,6 @@ service:
     initial_delay_seconds: 1200
   replica_policy:
     min_replicas: 2
-    auto_restart: true
```

Comment:
do we fully deprecate `auto_restart`?

Reply:
I thought we do according to @cblmemo. Could you confirm?

Reply:
@Michaelvll I remember so.
Maybe we can remove `auto_restart` from all our examples.
`auto_restart` is deprecated in this pr.
https://github.com/skypilot-org/skypilot/pull/2878

Reply:
Ahh, I see! If the PR has not been merged yet, maybe we can revert the change in this PR and have the change in the #2878 instead?

Reply:
Would vote for removing this arg from this example since we plan to advertise.

Reply:
In that case, people may see their replica fails, and SkyPilot fails to recover the replica, before #2878 is merged. I don't think that is expected? Can we create a smaller PR to deprecate `auto_restart` option and get it merged first before we merge #2878 to make sure the behavior of the example aligns with our description?